roc/roc-wrapper.go

```diff
@@ -1,19 +1,30 @@
 package roc
 
-import "fmt"
+import "errors"
 
-func convertErr(err int32, errstr string) error {
-	if err == 0 {
-		return nil
-	}
-	return fmt.Errorf("%s: %d", errstr, err)
-}
+var (
+	// ErrInvalidArguments indicates that one or more arguments passed to the function
+	// are invalid
+	ErrInvalidArguments = errors.New("One or more arguments are invalid")
+
+	// ErrInvalidApi should never happen and indicates that the API don't follow the declared contract
+	ErrInvalidApi = errors.New("Unknown error")
```

Comment:
I guess we should change error text ("Unknown error") as well.

Reply:
done